src/main/java/io/jenkins/plugins/neuvector/NeuVectorWorker.java

```diff
@@ -627,7 +627,7 @@ private void getToken(CloseableHttpClient httpclient) throws AbortException {
             }
             throw new AbortException(causeMessage);
         } catch (ClientProtocolException e) {
-            String causeMessage = "Invalid NeuVector controller IP or port.";
+            String causeMessage = "Invalid settings of the NeuVector Vulnerability Scanner on the global configuration page.";
```

Comment:
Does this exception related to the ControllerApiUrl?
if so, I think this error should be 
`Invalid settings of the controller api url in NeuVector Vulnerability Scanner on the global configuration page.`

Otherwise, this message looks ambiguous.